src/api/apiConstants.ts

```diff
@@ -0,0 +1,7 @@
+export const ApiConstants = {
+    Patient:{
+      GetAll:'Patients/GetPatients'
```

Comment:
can we please aensure code is not shared for review w/o proper foramtting, this has been mentioned multiple times across multiple PRs.